Makefile

```diff
@@ -16,15 +16,16 @@ GOMODULECMD    :=main
 
 
 SEMVER_MAJOR_VERSION    ?=3
-SEMVER_MINOR_VERSION    ?=0
-SEMVER_PATCH_VERSION    ?=1
+SEMVER_MINOR_VERSION    ?=1
+SEMVER_PATCH_VERSION    ?=0
```

Comment:
I would not define them. This will be too old in a few releases. What about checking their presence? 
```
ifndef SEMVER_MAJOR_VERSION
$(error SEMVER_MAJOR_VERSION is not set)
endif

```

Reply:
Hmm, that would mean that can't a simple `make` without parameters.

I've set the version to `0.0.0` to make it an obvious "dev" version, if you did not specify a real version. WDYT?